.template/addons/github/.github/workflows/test.yml.tt

```diff
@@ -82,15 +107,16 @@ jobs:
   system_tests:
     name: System tests
     runs-on: ubuntu-latest
-    needs: build
+    needs: linting
+    timeout-minutes: 15
     steps:
       - uses: actions/checkout@v3
 
       - name: Set env BRANCH_TAG
         uses: nimblehq/branch-tag-action@v1.2
 
       - name: Login to Docker registry
-        uses: docker/login-action@v1.6.0
+        uses: docker/login-action@v2
```

Comment:
Should we also replace the 2 other references to this action version? 

<img width="426" alt="image" src="https://user-images.githubusercontent.com/77609814/215940386-e1f8dc96-c823-4b4b-9be1-4bd5b4960064.png">

Reply:
(for consistency mainly)

Reply:
Sure, I can do that too

Reply:
Done in https://github.com/nimblehq/rails-templates/pull/377/commits/8a2bc6c208118470f3423100c5bd51d935dc6526